iidm/iidm-serde/src/main/java/com/powsybl/iidm/serde/NetworkSerDe.java

```diff
@@ -488,10 +489,22 @@ public static Anonymizer writeAndValidate(Network n, Path xmlFile) {
         return writeAndValidate(n, new ExportOptions(), xmlFile);
     }
 
+    /**
+     * Writes given network to JSON file, then reads the resulting file, then write the resulting network
+     * to the given path in XML format and validates the result with iidm .
+     */
     public static Anonymizer writeAndValidate(Network n, ExportOptions options, Path xmlFile) {
```

Comment:
This testing method should not be here as writeAndValidate and not intended to be only used for unit testing. It could be used by anyone willing to load and apply xml validation. So when used with Json format, we should just print a warning to inform that validation is not supported for json.
The roundtrip xml + json code should be in testing code (so in AbstractSerderTest)